src/main/java/org/jenkinsci/plugins/tuleap_git_branch_source/webhook/TuleapWebHook.java

```diff
@@ -0,0 +1,44 @@
+package org.jenkinsci.plugins.tuleap_git_branch_source.webhook;
+
+import hudson.Extension;
+import hudson.model.UnprotectedRootAction;
+import hudson.util.HttpResponses;
+import org.kohsuke.stapler.HttpResponse;
+import org.kohsuke.stapler.StaplerRequest;
+import org.kohsuke.stapler.StaplerResponse;
+
+import javax.annotation.CheckForNull;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+@Extension
+public class TuleapWebHook implements UnprotectedRootAction {
+
+    private static final Logger LOGGER = Logger.getLogger(TuleapWebHook.class.getName());
```

Comment:
This seems to be a dependency of the class, should not it be injected?

Reply:
No, whenever a class need a logger, it always declared like that.

Reply:
I'm a bit surprised by this. I'm not sure to understand why it is done this way but fine.